test/components/homepage/home.js

```diff
@@ -42,19 +45,21 @@ test('CUMULUS-799 Home page contains distribution error report', async (t) => {
   );
 
   const errorMetrics = home.find('#distributionErrors li');
-  t.is(errorMetrics.length, 2);
-  t.is(errorMetrics.at(0).key(), 'Gateway Execution Errors');
-  t.is(errorMetrics.at(1).key(), 'Gateway Access Errors');
-  t.is(errorMetrics.at(0).find('Link').dive().text(), `${tally(dist.apiGateway.execution.errors)} Gateway Execution Errors`);
-  t.is(errorMetrics.at(1).find('Link').dive().text(), `${tally(dist.apiGateway.access.errors)} Gateway Access Errors`);
+  t.is(errorMetrics.length, 3);
+  t.is(errorMetrics.at(1).key(), 'Gateway Execution Errors');
+  t.is(errorMetrics.at(2).key(), 'Gateway Access Errors');
+  t.is(errorMetrics.at(1).find('Link').dive().text(), `${tally(dist.apiGateway.execution.errors)} Gateway Execution Errors`);
```

Comment:
Generally the Kibana Links will be external, so change the fake/local/link to https:/fake.com and update these to find <a> instead of "Link"